drivers/fuzz/src/fuzz_target_lms.rs

```diff
@@ -0,0 +1,56 @@
+// Licensed under the Apache-2.0 license
+
+#![cfg_attr(feature = "libfuzzer-sys", no_main)]
+
+#[cfg(all(not(feature = "libfuzzer-sys"), not(feature = "afl")))]
+compile_error!("Either feature \"libfuzzer-sys\" or \"afl\" must be enabled!");
+
+#[cfg(feature = "libfuzzer-sys")]
+use libfuzzer_sys::fuzz_target;
+
+#[cfg(feature = "afl")]
+use afl::fuzz;
+
+use std::mem::size_of;
+
+use caliptra_drivers::{Lms, Sha256SoftwareDriver};
+use caliptra_lms_types::{LmsPublicKey, LmsSignature};
+
+pub const SHA256_DIGEST_WORD_SIZE: usize = 8;
+// TODO: When using SHA256_DIGEST_WORD_SIZE, these will accompany?
+pub const LOCAL_P: usize = 34;
+pub const LOCAL_H: usize = 5;
+
+fn harness(data: &[u8]) {
+    let input: &[u8];
+    let pub_key: &LmsPublicKey<SHA256_DIGEST_WORD_SIZE>;
+    let sig: &LmsSignature<SHA256_DIGEST_WORD_SIZE, LOCAL_P, LOCAL_H>;
+
+    // Note that this is a large minimum, but working in theory
+    if data.len() < (size_of::<LmsPublicKey<SHA256_DIGEST_WORD_SIZE>>() + size_of::<LmsSignature<SHA256_DIGEST_WORD_SIZE, LOCAL_P, LOCAL_H>>()) {
+        return;
+    }
+
+    // TODO: Alternatively, use structure-aware fuzzing, input comprising arguments
+    unsafe {
+        input = &data[(size_of::<LmsPublicKey<SHA256_DIGEST_WORD_SIZE>>() + size_of::<LmsSignature<SHA256_DIGEST_WORD_SIZE, LOCAL_P, LOCAL_H>>())..];
+        pub_key = &*(data.as_ptr() as *const LmsPublicKey<SHA256_DIGEST_WORD_SIZE>);
+        sig = &*(data[size_of::<LmsPublicKey<SHA256_DIGEST_WORD_SIZE>>()] as *const LmsSignature<SHA256_DIGEST_WORD_SIZE, LOCAL_P, LOCAL_H>);
+    }
```

Comment:
nit: Could use zerocopy to copy the data into a rust struct. Would make this code easier to read (and non-unsafe). But this may not be relevant if you switch to structure-aware

Reply:
The `input` argument to `verify_lms_signature()` is itself a byte slice, and this doesn't implement FromBytes - possibly odd, since the implementation is trivial, I believe? I'm not sure if it would be possible to zerocopy the `data` argument here to the struct we need.

Reply:
I mean something like this:

```rust
let mut offset = 0;
let pub_key = ImagePublicKey::read_from_prefix(data);
offset += sizeof::<ImagePublicKey>();
let sig = ImageLmsSignature::read_from_prefix(&data[offset..];
offset += sizeof::<ImageLmsSignature>();
let input = &data[offset..];
```

This would require deriving FromBytes for those types, but I think that's ok.

Reply:
Ah, I see. `read_from_prefix()` will work.